packages/suite/src/components/wallet/AccountTopPanel/components/AccountNavigation/index.tsx

```diff
@@ -1,30 +1,106 @@
 import React from 'react';
-import { ITEMS } from '@wallet-config/menu';
 import { Account } from '@wallet-types';
-import AppNavigation from '@suite/components/suite/AppNavigation';
+import AppNavigation, { AppNavigationItem } from '@suite/components/suite/AppNavigation';
+import { Translation } from '@suite-components/Translation';
+import { useActions } from '@suite-hooks';
+import * as routerActions from '@suite-actions/routerActions';
+import * as modalActions from '@suite-actions/modalActions';
 
 interface Props {
     account?: Account;
+    filterPosition?: 'primary' | 'secondary';
+    dataTestSuffix?: string;
+    primaryContent?: React.ReactNode;
 }
 
 const AccountNavigation = (props: Props) => {
     const { account } = props;
+    const { goto, openModal } = useActions({
+        goto: routerActions.goto,
+        openModal: modalActions.openModal,
+    });
+    const ITEMS: AppNavigationItem[] = [
+        {
+            id: 'wallet-index',
+            callback: () => {
+                goto('wallet-index', undefined, true);
+            },
+            title: <Translation id="TR_NAV_TRANSACTIONS" />,
+            position: 'primary',
+            isHidden: () => false,
+        },
+        {
+            id: 'wallet-details',
+            callback: () => {
+                goto('wallet-details', undefined, true);
+            },
+            title: <Translation id="TR_NAV_DETAILS" />,
+            position: 'primary',
+            isHidden: () => account?.networkType !== 'bitcoin',
+        },
+        {
+            id: 'wallet-send',
+            callback: () => {
+                goto('wallet-send', undefined, true);
+            },
+            title: <Translation id="TR_NAV_SEND" />,
+            position: 'secondary',
+            isHidden: () => false,
+        },
+        {
+            id: 'wallet-receive',
+            callback: () => {
+                goto('wallet-receive', undefined, true);
+            },
+            title: <Translation id="TR_NAV_RECEIVE" />,
+            position: 'secondary',
+            isHidden: () => false,
+        },
+        {
+            id: 'wallet-coinmarket-buy',
+            callback: () => {
+                goto('wallet-coinmarket-buy', undefined, true);
+            },
+            title: <Translation id="TR_NAV_TRADE" />,
+            position: 'secondary',
+            isHidden: () => false,
```

Comment:
hm, idk why but this button is rendered as primary:
![Screenshot from 2021-04-06 09-24-08](https://user-images.githubusercontent.com/3435913/113673966-eaf56900-96b9-11eb-9e59-13ea154ee60f.png)

Reply:
zeplin seems to be outdated, resolving